### ISSUE TITLE
Mismatch between spec and implementation with inline contextual multiple substitution rules

### ISSUE THREAD

**Issue by cmyr** [adobe-type-tools/afdko#1447]
[The spec says](https://github.com/adobe-type-tools/afdko/blob/bf219ceb7d10deb1a8d85c5df04e98e3b48025fc/docs/OpenTypeFeatureFileSpecification.md?plain=1#L1775-L1779):

> If there is only a single substitution operation, and it is either single substitution or ligature substitution, then the operation can be specified in-line and its type will be auto-detected from the input and replacement sequence in the same way as in their corresponding standalone (i.e. non-contextual) statements.

however the implementation will accept the following statement with an inline multiple substitution rule, and will generate a GSUB type 2 table for it.

```fea
feature hmmm {
    sub a b' by c d;
} hmmm;
```

This is not a big issue, but wanted to make sure it was intentional.

**Comment by josh-hadley** [comment]
I suspect this is a simple oversight and the spec text should probably read:
> \[...\] and it is a single, multiple, or ligature substitution \[...\]

I'll do some research here and try to confirm that.

**Comment by cmyr** [comment]
I agree that is the most likely answer (and the easiest one to fix 😅)

**Comment by josh-hadley** [comment]
My research suggests this is the case -- I chatted with several folks here and consensus seems to be that the original text was probably written prior to `makeotfexe` supporting GSUB lookupType2, and when that support was added the spec text was never updated.

On @frankrolf's suggestion I also checked to see what [`fontmake`](https://github.com/googlefonts/fontmake) does, and it behaves exactly the same as AFDKO/makeotfexe  (a Type6/format 1 context lookup that references a Type2 that performs the `b` -> `c d` substitution).
